homeassistant/components/tractive/entity.py

```diff
@@ -15,7 +18,10 @@ class TractiveEntity(Entity):
     _attr_has_entity_name = True
 
     def __init__(
-        self, user_id: str, trackable: dict[str, Any], tracker_details: dict[str, Any]
+        self,
+        client: TractiveClient,
+        trackable: dict[str, Any],
+        tracker_details: dict[str, Any],
```

Comment:
```suggestion
        tracker_details: dict[str, Any],
        dispatcher_signal: str = f"{TRACKER_HARDWARE_STATUS_UPDATED}-{self._tracker_id}"
```

To remove the assert below, you can just pass in the dispatcher signal.

Reply:
Hmmm I think `self._tracker_id` is not accessible here.

Reply:
And we would have to pass `dispatcher_signal` from device tracker platform where we use two different dispatcher signals.

Reply:
I did one more approach to the case and managed to remove the assert.